### ISSUE TITLE
long comment or multiline string at end of file isn't ignored properly

### ISSUE THREAD

**Issue by toejough** [PyCQA/pylint#2343]
### Steps to reproduce
1.  put a long multiline string or a long comment at the end of a file
2. add `line-too-long` to the list of ignored errors in `.pylintrc`
3. run `pylint <file>`

### Current behavior
`line-too-long` error is returned

### Expected behavior
`line-too-long` error should be ignored

### pylint --version output
```shell
> pylint --version
pylint 2.0.1
astroid 2.0.1
Python 3.6.5 (default, May 11 2018, 09:14:55)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```
Repro'd on 2.0.0 and 2.0.1 releases, the pre-release version as specified in the issue template, as well as master (as of https://github.com/PyCQA/pylint/commit/2b24ec063a5355648d286633e37f084ced4f1604)

### Examples
long final comment:
```python
"""Testing long lines."""


print("requires some code before the comment")
# Some really long comment.  No, no, no - MUCH longer than this.  At least 100 chars.  Keep going.  Yeah,
about this long works.
```

really, it's a long comment that isn't followed by more code - it doesn't have to be the last comment:
```python
"""Testing long lines."""


print("requires some code before the comment")
# Some really long comment.  No, no, no - MUCH longer than this.  At least 100 chars.  Keep going.  Yeah,
about this long works.
# The long comment is still caught even if it's not the last line.
```

Long multi-line strings work, too:
```
"""Testing long lines."""


print(
    f"Some really long string.  No, no, no - MUCH longer than this.  At least 100 chars.  Keep going.  Yeah, about this long works."
    f"Some really long string.  No, no, no - MUCH longer than this.  At least 100 chars.  Keep going.  Yeah, about this long works."
)
```

### Similar issue
I found #1950 which was talking about this same problem.  There was investigation, acknowledgement, a fix proposed and merged, and then the issue was closed.

Then it looks like the merged build failed, and the fix was reverted, but the issue was never re-opened.

**Comment by brycepg** [comment]
Hi, looks like my PR #2342 fixes this as well
